.github/workflows/main.yml

```diff
@@ -353,6 +353,9 @@ jobs:
       - name: Check for duplicate Node dependencies
         run: yarn dedupe --check
 
+      - name: Check for CJS requires which could be ESM imports
+        run: node ./tools/cjs-to-esm-candidates.mjs --check
+
```

Comment:
Thoughts on running this check in CI? This has the potential to break CI on WIP PRs, but I think it's worth it to force folks to match our expectations before they can merge.

Reply:
I'm in favour.